src/GovUk.Education.ExploreEducationStatistics.Admin/Services/DataArchiveValidationService.cs

```diff
@@ -56,6 +58,17 @@ public DataArchiveValidationService(IFileTypeService fileTypeService)
             var dataFile = file1.Name.Contains(".meta.") ? file2 : file1;
             var metaFile = file1.Name.Contains(".meta.") ? file1 : file2;
 
+            var filenamesValid = ValidateFilenameLengths(
+                zipFile.FileName.Length,
+                dataFile.Name.Length,
+                metaFile.Name.Length,
+                out var isValid);
+
+            if (!isValid)
```

Comment:
Not a big deal, but you can use `filenamesValid.IsLeft` here instead of `!isValid`, and so remove the need for an out param.